compiler/src/dotty/tools/dotc/transform/ElimRepeated.scala

```diff
@@ -92,7 +93,7 @@ class ElimRepeated extends MiniPhase with InfoTransformer { thisPhase =>
       JavaSeqLiteral(elems, elemtpt)
     case _ =>
       val elemType = tree.tpe.elemType
-      var elemClass = elemType.classSymbol
+      var elemClass = erasure(elemType).classSymbol
       if (defn.NotRuntimeClasses.contains(elemClass)) elemClass = defn.ObjectClass
```

Comment:
Is this line still needed if we erase the `elemType`?

Reply:
I am not sure about NullClass, NothingClass. I think these survive erasure.